frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -760,19 +761,21 @@ private void prepareDetailView(RenderRequest request, RenderResponse response) {
     private void setUsingDocs(RenderRequest request, User user, ComponentService.Iface client, Set<String> releaseIds) {
         Set<Project> usingProjects = null;
         Set<Component> usingComponentsForComponent = null;
+        int allUsingProjectsCount = -1;
```

Comment:
Why is `-1` the fallback, instead of `0`?

Reply:
instead of using `-1` as fallback one could only fill it after line 769, i.e. if the call was successful

Reply:
Oh, now I see https://github.com/sw360/sw360portal/pull/808/files#diff-e8bd4c4dc1849fc1be46b0781739dcd5R13. I think that this is not the nicest way of handling exceptions.

Reply:
`usingProjectTable` evalutes the value of `allUsingProjectsCount`.
You are right, its not the best way of handling exceptions. 
`setUsingDocs` should throw the exception instead of only logging it. 
But I dont want to refactor this method here.